modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/SettingsImpl.kt

```diff
@@ -555,6 +555,12 @@ class SettingsImpl @Inject constructor(
         sharedPrefs = sharedPreferences,
     )
 
+    override val suggestedFolderPaywallDismissTime = UserSetting.LongPref(
+        sharedPrefKey = "globalAutoDownloadNewEpisodes",
```

Comment:
The key for this doesn't look right.

Reply:
Copy and paste issue 🤦🏽‍♀️

It's fixed now [a0934fc](https://github.com/Automattic/pocket-casts-android/pull/3573/commits/a0934fc32425bef61dc760d75d58aef582d1dcda)

Thanks for catching this!